client/dashboard/app/queries/reader-teams.ts

```diff
@@ -0,0 +1,8 @@
+import { fetchReaderTeams, ReaderTeam } from '../../data/reader-teams';
+
+export const isA8CTeamMemberQuery = () => ( {
```

Comment:
What do you think of simply:

- `reader-teams.ts` => `a8c.ts`
- `isA8CTeamMemberQuery` => `isAutomatticianQuery`
- `isA8CTeamMember` => `isAutomattician`

? 🥲